CHANGELOG.rst

```diff
@@ -1,5 +1,73 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This release contains some very nice improvements to the file and datasets management and visualization as well as project migrations.
+
+New features
+~~~~~~~~~~~~
+
+- **Dataset removal** A dataset can be now removed from a project from the UI. Note that if a removed dataset was part of a project with the Knowledge Graph activated, its metadata will not be removed.
```

Comment:
I would remove this part...  Note that if a removed dataset was part of a project with the Knowledge Graph activated, its metadata will not be removed. 
 Since datasets can't be accessed anymore, from a user's perspective the datasets are gone.